azure.js

```diff
@@ -8,6 +8,8 @@ if (!process.env['AZURE_SUBSCRIPTION_ID']) {
   throw new Error('Please set the AZURE_SUBSCRIPTION_ID environment variable')
 }
 const subscriptionId = process.env['AZURE_SUBSCRIPTION_ID']
+let intervalHandle = null;
```

Comment:
Should the `setInterval()` call at the end (line 179) also have its result assigned to `intervalHandle`?